docs/tokens/nft-tokens.md

```diff
@@ -596,13 +596,21 @@ TransferTransaction {
     Value: 0
     GasLimit: 1000000 + length of Data field in bytes * 1500
     Data: "ESDTNFTTransfer" +
-          "@" + <token identifier in hexadecimal encoding> +
-          "@" + <the NFT nonce in hexadecimal encoding> +
+          "@" + <collection identifier in hexadecimal encoding> +
+          "@" + <the NFT nonce> +
           "@" + <quantity to transfer in hexadecimal encoding> +
           "@" + <destination address in hexadecimal encoding>
 }
 ```
 
+:::tip
+Here is an example of an NFT identifier: `ABC-1a9c7d-05dc`
+
+The collection identifier is `ABC-1a9c7d` and the NFT nonce is `05dc`
+
+Also note that an Elrond address is in bech32 so you will need to convert the address from bech32 to hexadecimal.
```

Comment:
This isn't enough. One might think that hexEncode(erd1...) will be enough. Further explanations will be added soon by us, but until then, at least mention this utility class from erdjs:
https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/address.ts

or, manually, use this external tool: https://slowli.github.io/bech32-buffer/  (Data tab, tag = `erd`)

Reply:
For the external tool, I added a direct link to the bech32 to hex of your converter, I find it easier for the person instead of telling him to go to the data tab then add the `erd` tag and then click on decode.

But I can change the link to : https://elrond-converters.netlify.app/ if you want